### ISSUE TITLE
Missing latest versions for mega-linter-runner v7 with npx

### ISSUE THREAD

**Issue by VictorRos** [oxsecurity/megalinter#2733]
**Describe the bug**

Missing latest versions for `mega-linter-runner` `v7` with `npx`.

I'm trying to migrate one of our repositories from `6.20.1` to `7.0.4` to validate migration steps for other repositories.

To be sure we are always using the same version for runner and docker image, we are using this command:

```shell
npx -y mega-linter-runner@6.20.1 --release v6.20.1 --flavor javascript --containername mega-linter
```

We do that because we had some unwanted behaviors when we were not clarifing version on 3 digits.
| mega-linter-runner version |      Working?      |
| -------------------------- | :----------------: |
| `latest`                   | :white_check_mark: |
| `7`                        | :white_check_mark: |
| `7.0.3`                    |        :x:         |
| `7.0.4`                    |        :x:         |

**To Reproduce**

1. Run following command

   ```shell
   npm view mega-linter-runner versions
   ```

2. Only these versions are available:

   ```shell
   [
     ...
     '7.0.0',
     '7.0.5-beta202306080448.0',
     '7.0.5-beta202306080735.0'
   ]
   ```

**Expected behavior**

I'm expecting to be able to use `7.0.3` or `7.0.4` version like `6.20.1`.

| mega-linter-runner version |      Working?      |
| -------------------------- | :----------------: |
| `latest`                   | :white_check_mark: |
| `7`                        | :white_check_mark: |
| `7.0.3`                    | :white_check_mark: |
| `7.0.4`                    | :white_check_mark: |

**Screenshots**

![image](https://github.com/oxsecurity/megalinter/assets/6940422/7ef41ace-20c6-46e5-82f6-32a240253758)

**Additional context**

In results, we can see `7.0.5-beta` versions.

May I assume it will be fixed for `7.0.5` and laters?

Regards,

Victor

**Comment by echoix** [comment]
I have no immediate answer, but to have all info available, what is the node version that is used or available in the system in question?

**Comment by echoix** [comment]
I see in the repo's package.json that the version is supposed to be 7.0.4. There is 7.0.0 on npmjs. There are also two 7.0.5 betas of yesterday published, but not others.
![image](https://github.com/oxsecurity/megalinter/assets/27212526/8efc0aa8-44fe-4115-acbb-4c9a72e7a7a0)

**Comment by VictorRos** [comment]
> I have no immediate answer, but to have all info available, what is the node version that is used or available in the system in question?

NodeJS version: `16.19.1`
NPM version: `8.19.4`

On my machine (MacOS M1).

Maybe a problem when publishing the package to `npmjs.com`?

**Comment by nvuillam** [comment]
There has been a npm internal error that prevented to publish new versions since 7.0.0

It took a while for support team to find a workaround, but it seems that they found it and that we can publish again (as the recent beta shows)

So nexxt minor version of MegaLinter (I'll try to publish it today), this problem will be solved :)

**Comment by nvuillam** [comment]
@VictorRos @echoix solved with v7.1.0 , just released

It seems the package has correctly been published :)

![image](https://github.com/oxsecurity/megalinter/assets/17500430/5e6a601d-66c8-4474-beef-9f4c595dec46)

**Comment by VictorRos** [comment]
Ok good news, thanks for letting me known.
